docs/photoniq/vwrs/03-configuration-and-setup/04-customizing-vwrs.md

```diff
@@ -0,0 +1,97 @@
+---
+sidebar_position: 30
+title: Customizing VWRs
+---
+
+## Fingerprinting
+
+VWRs can use device fingerprinting techniques to help mitigate cookie sharing and replay attacks. By default, a basic fingerprint is generated using information derived from the device object ([https://techdocs.akamai.com/edgeworkers/docs/device-object](https://techdocs.akamai.com/edgeworkers/docs/device-object)). If stronger device fingerprinting is required, then you can override this logic with your own, or configure it to use one of the fingerprinting techniques available in Akamai’s security products.
+
+## Priority
+
+VWRS allows you customize a waiting room with up to ten priorities, with these priority numbers being positive integers from 1 through 255. 
+
+### Configuring Priority
+
+Configure the priorities using the `POST/PUT/PATCH /api/vwr/v1/domains` REST API.
+
+For instance, to configure three priorities with values of 1, 20, and 255, use the `PATCH /api/vwr/v1/domains` REST API to update the domain:
+
+```bash
+curl -X 'PATCH' \
```

Comment:
The text above says it's using the `domains` (renamed to `waitingrooms`) API, but the example here is calling the `origins` API, which has been deleted 😮 

@maryann-macrometa Please point me to where you got this example from, so that I can check what it has in mind and fix it accordingly.

Reply:
@todor-macrometa its part of the current docs . Here, https://www.macrometa.com/docs/photoniq/vwrs/configure-vwrs-edgeworker